### PR TITLE
update policy to ref stat software; closes #446

### LINKED ISSUE #446: Add other mention of stat software review
In https://devguide.ropensci.org/policies.html#other-scope-considerations

> Here are some types of packages we are unlikely to accept:

>    Packages that wrap or implement statistical or machine learning methods. We are not organized so as to review the correctness of these methods (But see “text analysis”, above).

### PR DESCRIPTION
**Pull request by mpadge**

### CODE REVIEW

File: softwarereview_policies.Rmd [hunk 1]
```diff
@@ -97,12 +95,6 @@ Packages should be *general* in the sense that they should solve a problem as br
 
 Packages that include interactive tools to facilitate researcher workflows (e.g., shiny apps) must have a mechanism to make the interactive workflow reproducible, such as code generation or a scriptable API.
 
-Here are some types of packages we are unlikely to accept:
-
--  Packages that wrap or implement statistical or machine learning methods.  We are not organized so as to review the correctness of these methods (But see "text analysis", above).
--  Exploratory data analysis packages that visualize or summarize data.
```
Comment (maelle) [inline softwarereview_policies.Rmd:103]: I'd still keep the two last examples?

**Review by maelle — CHANGES_REQUESTED** [review]
Thank you!

File: softwarereview_policies.Rmd (hunk 1 above)
Comment (mpadge) [reply softwarereview_policies.Rmd:103]: But they're both stats packages and so can be accepted?

Comment (maelle) [reply softwarereview_policies.Rmd:103]: ooooh

**Review by maelle — APPROVED** [review]
Thank you for the PR & patient clarifications :pray:

**Pull request merged**